mobile/native/src/trade/order/handler.rs

```diff
@@ -17,9 +17,12 @@ pub async fn submit_order(order: Order) -> Result<()> {
     let url = format!("http://{}", config::get_http_endpoint());
     let orderbook_client = OrderbookClient::new(Url::parse(&url)?);
 
-    if let Err(e) = position::handler::update_position_after_order_submitted(order) {
-        order_failed(Some(order.id), FailureReason::OrderNotAcceptable, e)?;
-        bail!("Could not submit order because extending/reducing the position is not part of the MVP scope");
+    let position = position::handler::get_current_position()?;
```

Comment:
🤔 Can we be sure that there is always a current position? I think we are still deleting a position from the app once it gets closed, don't we? If yes, that will fail whenever we are opening a position?

Reply:
was it? in the previous call we were handling if the was a current position. This is simply failing if not position is found. Or am I reading this wrong?

```
if let Some(position) = db::get_positions()?.first() {
```

Reply:
If we are unable to submit the order to the orderbook we should set it to `Failed`. 
This logic has been removed now - I don't think this is intended.